_tags

```diff
@@ -1,4 +1,4 @@
 true : annot, warn(-P)
 "src/core" : include
 "src/support" : include
-<**/*.{byte,native}> : package(unix, extlib, ulex, camlp4.lib, str), use_core, use_support
+<**/*.{byte,native}> : package(unix, extlib, sedlex, gen, str)
```

Comment:
likewise, `sedlex`? (though I'm not sure it is really unneeded or not)

Reply:
`_tags` files were used by ocamlbuild, which we aren't using anymore. I'll amend the PR to remove all of them.